Python/samples/cartpole/main.py

```diff
@@ -387,6 +388,18 @@ def CreateSession(
                 episode += 1
             elif event.type == "EpisodeStep":
                 iteration += 1
+                if sim_speed > 0:
+                    if sim_speed_variance > 0:
```

Comment:
The variance of a random uniform is $\frac{1}{12}(\max U - \min U)^2$, not what is defined here, which might be confusing? Perhaps it'll be easier to just provide a single argument, say `sim_delay`, and then if `sim_delay > 0` then do `random.uniform(0, sim_delay)`.

Reply:
yeah, i guess if you want the fixed delay then you would need two parameters. I guess if so just note that `sim_speed_variance` isn't actually variance as defined here :)

Reply:
Hi @akzaidi thanks for your comment. I've changed the distribution to be a truncated normal distribution so that the variance is more intuitive.

- I've updated the readme (now with a distribution pic)
- Tested the code for `--sim-speed 3 --sim-speed-variance 1`
- colleted the logs
- plotted the resulting dist to double check